frontend/src/host_orchestrator/orchestrator/instancemanager.go

```diff
@@ -347,7 +343,7 @@ type startCVDParams struct {
 	BootloaderDir string
 }
 
-func (h *startCVDHandler) Create(p startCVDParams) error {
+func CreateCVD(execCtx cvd.CVDExecContext, p startCVDParams) error {
```

Comment:
nit: `ctx`. 

`ctx` is pretty common in Golang for contexts instances, and in this function we don't have any other context variable to differentiate from.

Reply:
Done